packages/contracts/contracts/Passport.sol

```diff
@@ -4,11 +4,15 @@ pragma solidity ^0.8.4;
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
 
 /**
  * The Passport contract is an ERC721 that represents member access for DAOs.
  */
 contract Passport is ERC721, Ownable, AccessControlEnumerable {
+  using Counters for Counters.Counter;
```

Comment:
Decided to use Counters here instead of generating tokenId on UI. This way, in case of airdrops we dont have to generate tokenIds for all tokens in the UI.

Reply:
this costs significantly more gas for work that we could do off chain. I think any tokenId generation we should do off chain unless there's some security benefit we gain from to justify the gas cost on chain.

If the concern is to avoid generating in the UI, we could generate them in the API layer

Reply:
done...generating token IDs on UI now